Dockerfile.cpu

```diff
@@ -0,0 +1,17 @@
+# Docker image for running examples in Tensorflow models.
```

Comment:
I don't see in Github's interface right now where these files will be placed in the repo-- is that because they are at the top level? If so, can we nest these somewhere under official/testing, maybe with an appropriate subdir there as well?

Reply:
Yes, it is at the top level, which is a common practice for defining container images.
Is there a reason you want it to under `testing`? I'd expect users to use these images even outside of testing as long as the model is stable and well tested.

Reply:
"As long as the model is stable and well tested" is an unsupported claim in this repo under the Research directory. Which means that, for example, research models may require other builds of TF, or released TF, etc. I would generally prefer to keep these under Official, to avoid the impression that we are supporting these for the research models. As to in the testing dir, more stylistic-- I like things to be contained and orderly ;) Especially if there will be many versions of these (ie, GPU/CPU/multi-GPU, nightly/stable, etc.